### PR TITLE
Edited PythonAPIOverview.md

### PR DESCRIPTION
**Pull request by AlexMuresan**
The example given in the "Creating an ONNX Model Using Helper
Functions" example.

### CODE REVIEW

File: docs/PythonAPIOverview.md [hunk 1]
```diff
@@ -66,25 +66,26 @@ from onnx import AttributeProto, TensorProto, GraphProto
 
 # Create one input (ValueInfoProto)
 X = helper.make_tensor_value_info('X', TensorProto.FLOAT, [1, 2])
+Z = helper.make_tensor_value_info('Z', TensorProto.FLOAT, [1, 2])
```
Comment (linkerzhang) [inline docs/PythonAPIOverview.md]: Thank you very much for improving the tutorials!

Do you mind to add comment in this doc to clarify the operator set versioning please? say, this is based on Pad-2 or maybe Pad-5.

**Comment by linkerzhang** [comment]
This is about issue #2490

File: docs/PythonAPIOverview.md (hunk 1 above)
Comment (linkerzhang) [reply docs/PythonAPIOverview.md]: otherwise, the same confusion will happen when Pad updated in future again.

Comment (AlexMuresan) [reply docs/PythonAPIOverview.md]: Ok. I've done the necessary changes and created a new pull request that seems to have added to this one. Hope everything is ok.

Thank you for the input!

**Comment by linkerzhang** [comment]
or make_model API should be used in this tutorial :).

**Comment by AlexMuresan** [comment]
@linkerzhang will look into it. Should I make a new pull request when I complete the changes or should I just ammend this commit?

**Comment by linkerzhang** [comment]
a new PR is good. thank you!

**Pull request merged**